transports/file/src/main/resources/plugin.properties

```diff
@@ -1,5 +1,6 @@
 #Plugin Properties
 #TODO MULE-10735 generate this file from maven dependencies so we won't have issues with the hard-coded version, packages or resources there
+plugin.dependencies=mule-module-http-ext-4.0-SNAPSHOT
```

Comment:
Because mule-transport-module-support depends on HTTP extension (for some constants needed that would be later added to the HTTP service API), therefore as all the transports depends on this support module they require to have this plugin dependency declared in order to access those classes.